daemon/cmd/kube_proxy_replacement.go

```diff
@@ -642,11 +642,7 @@ func finishKubeProxyReplacementInit(isKubeProxyReplacementStrict bool) {
 		}
 	}
 
-	option.Config.NodePortHairpin = len(option.Config.Devices) == 1
-	if option.Config.NodePortAcceleration != option.NodePortAccelerationDisabled &&
-		len(option.Config.Devices) != 1 {
-		log.Fatalf("Cannot set NodePort acceleration due to multi-device setup (%q). Specify --%s with a single device to enable NodePort acceleration.", option.Config.Devices, option.Devices)
-	}
+	return nil
```

Comment:
No need to return nil.

Reply:
Hm, if I remove that line, then the commit is empty. Do you still need this PR to be backported?

Reply:
I've force pushed a new version with the line deletions and without the line addition.

Reply:
:+1: Ah thanks, I must have gotten confused with the conflict resolution.